### PR TITLE
Setup CircleCi Test Summary

### PR DESCRIPTION
**Pull request by user512**
https://github.com/papertrading/PaperTrading/issues/28

Set up CI test summary
Fix existing failing spec
Use pry byebug, remove web console and better error

### CODE REVIEW

**Comment by user512** [comment]
screen shot 2016-10-29 at 11 09 19 am
Test Summary is available now on CI 😄

File: app/models/new_york_time.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 class NewYorkTime
+  BUSINESS_NEWS_ENDPOINT='https://api.nytimes.com/svc/topstories/v2/business.json?api-key='
 
   def self.get_business_news
-    response = HTTParty.get("https://api.nytimes.com/svc/topstories/v2/business.json?api-key=#{ENV['NY_KEY']}")
+    response = HTTParty.get("#{BUSINESS_NEWS_ENDPOINT}#{ENV['NY_KEY']}")
     data_to_json = JSON.parse(response.body, symbolize_names: true)[:results]
     # sort news array [latest ... oldest]
     data_to_json.sort { |data1, data2| Time.parse(data2[:created_date]).to_i <=> Time.parse(data1[:created_date]).to_i }
```
Comment (user512) [inline app/models/new_york_time.rb:8]: Instead of using command to explain this logic, we can extract it into a private method and name it as `sort_news`.

Comment (user512) [reply app/models/new_york_time.rb:8]: I also noticed the test is only testing if news is `nil`, perhaps we should be testing if it's parsing and sorting the news correctly.
